filebeat/tests/system/test_registrar.py

```diff
@@ -269,7 +269,7 @@ def test_registry_file_update_permissions(self):
             max_timeout=1)
 
         # Wait a moment to make sure registry is completely written
-        time.sleep(1)
+        time.sleep(10)
```

Comment:
Do we really need to wait 10s here?

Can we replace the next assertion for something like `self.wait_until` that will keep trying until the condition is true or the timeout is reached?

That will help the tests to run faster.